packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1619,3 +1623,272 @@ Multiple IdP instances can be defined in a single Policy Action. This allows use
+
+> **Note:** You can have a maximum of 5000 app sign-on policies in an org.
+> There is a max limit of 100 rules allowed per policy.
+> When you create an app sign-on policy, you will automatically also create a default policy rule with the lowest priority of `99`.
+> The highest priority that an app sign on policy rule can be set is `0`.
+
+
+#### App Sign On Policy example
+
+```json
+    {
+        "type": "ACCESS_POLICY",
+        "name": "Web Cart App Sign On Policy",
+        "description": "Standard policy for Web Cart application"
+    }
+```
+
+### Policy conditions
+
+Policy conditions are not supported, conditions are applied at the rule level for this type of policies.
+
+### Policy Rules conditions
+
+The following conditions may be applied to the rules associated with an app sign on policy:
+
+* [People Condition](#people-condition-object)
+
+* [Network Condition](#network-condition-object)
+
+* [Device Condition](#device-condition-object)
+
+* [Platform Condition](#platform-condition-object)
+
+* [Expression Language Condition](#el-condition-object)
+
+* [Office365Client Condition](#office365Client-condition-object)
+
+* [Risk Condition](#riskscore-condition-object)
+
+* [User Type Condition](#usertype-condition-object)
+
+#### App Sign On Action default example
+
+```json
+  "actions": {
+        "appSignOn": {
+            "access": "DENY",
+            "verificationMethod": {
+                "factorMode": "1FA",
+                "type": "ASSURANCE",
+                "reauthenticateIn": "PT43800H"
+            }
+        }
+    }
+```
+
+#### App Sign On Action object
+
+| Property                | Description                                                                                                                                                               | Data Type                                       | Required                      | Default |
+| ---                     | ---                                                                                                                                                                       | ---                                             | ---                           | ---     |
+| `access`                  | `ALLOW` or `DENY`                                                                                                                                                         | `ALLOW` or `DENY`                               | Yes                           | N/A     |
+| `verificationMethod`      | A Verification Method describes the means by which the user must be verified. The only supported type is `ASSURANCE`.                                                       | Object                                          | Yes                           | [Default](#app-sign-on-action-default-example)        |
+
+
+### Verification Method Object
+
+A Verification Method describes the means by which the user must be verified. The only supported type is `ASSURANCE`.
```

Comment:
change to? `The verification method ensures that user must be verified. Only supported type is `ASSURANCE``

Reply:
I have made the changes.